lib/ansible/modules/windows/win_domain.ps1

```diff
@@ -75,6 +76,7 @@ try {
     $forest_context = New-Object -TypeName System.DirectoryServices.ActiveDirectory.DirectoryContext -ArgumentList Forest, $dns_domain_name
     $forest = [System.DirectoryServices.ActiveDirectory.Forest]::GetForest($forest_context)
 } catch [System.DirectoryServices.ActiveDirectory.ActiveDirectoryObjectNotFoundException] {
+    Add-Warning -obj $result -message "Error during forest query initialization: $($_.Exception.Message)"
```

Comment:
I wouldn't add this warning, it just means there wasn't a forest that was found which is expected in some situations.